packages/modal/useModalFocus.ts

```diff
@@ -11,7 +11,7 @@ export const useModalFocus = (): React.RefObject<HTMLDivElement> => {
 
     return Array.from(nodes).filter(
       (node): node is HTMLElement =>
-        node && node instanceof HTMLElement && node.offsetParent !== null
+        node && node instanceof HTMLElement && node.offsetParent !== null,
```

Comment:
```suggestion
        node instanceof HTMLElement && node.offsetParent !== null,
```

Reply:
Makes sense, will update once I get to this hook